internal/provider/resource_site.go

```diff
@@ -0,0 +1,102 @@
+package provider
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/paultyng/go-unifi/unifi"
+)
+
+func resourceSite() *schema.Resource {
+	return &schema.Resource{
+		Description: "`unifi_site` manages Unifi sites",
+
+		Create: resourceSiteCreate,
+		Read:   resourceSiteRead,
+		Update: resourceSiteUpdate,
+		Delete: resourceSiteDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"description": {
```

Comment:
Need to override the default ID here and then regen the docs since this isn't actually optional.

```suggestion
			"id": {
				Description: "The ID of the site.",
				Type:            schema.TypeString,
				Computed:   true,
			},
			"description": {
```